packages/remix-dev/config/defaults/entry.server.node.tsx

```diff
@@ -47,11 +47,12 @@ function handleBotRequest(
         onAllReady() {
           shellRendered = true;
           const body = new PassThrough();
+          const webBody = Readable.toWeb(body) as ReadableStream;
```

Comment:
This cast seems to be required due to the underlying generic, but I couldn't find a way to get them to align without the `as`.

```
Type 'ReadableStream<T>' is missing the following properties from type 'ReadableStream<ArrayBufferView | undefined>': 
  values, [Symbol.asyncIterator]ts(2345)
```